etc/profile-m-z/w3m.profile

```diff
@@ -28,12 +31,14 @@ include disable-exec.inc
 include disable-interpreters.inc
 include disable-programs.inc
 include disable-shell.inc
-include disable-x11.inc
```

Comment:
Can it also open images on wayland?

If so, remove `blacklist ${RUNUSER}/wayland-*` as well.